codingTracker.jakubmikolajewski/CodingTrackerLibrary/CodingController.cs

```diff
@@ -0,0 +1,279 @@
+﻿using Dapper;
+
+namespace CodingTrackerLibrary
+{   
+    public class CodingController
+    {
+        public static Dictionary<string, string> valuesToUpdate = [];
+        public static void SelectAllSessionsQuery()
+        {        
+            string command = $"SELECT * FROM 'Coding sessions';";
+
+            DatabaseActions current = new DatabaseActions();
+            using (current.Connection)
+            {
+                try
+                {
+                    using (var reader = current.Connection.ExecuteReader(command))
+                    {
+                        while (reader.Read())
+                        {
+                            CodingSession session = new CodingSession(reader.GetInt32(0), reader.GetString(1), reader.GetString(2), reader.GetInt32(3));
```

Comment:
The beauty of Dapper is that you no longer have to use a Reader object and all this horrible boilerplate code. This can be simplified to:
```
var sessions = current.Connection.Query<CodingSession>(command).ToList();
TableVisualizationEngine.selectAllSessions.AddRange(sessions);
```